runners/org.osbuild.rhel82

```diff
@@ -49,7 +49,6 @@ def python_alternatives():
         pass
```

Comment:
```
************* Module org.osbuild
runners/org.osbuild.rhel82:6:0: W0611: Unused import osbuild.api (unused-import)
runners/org.osbuild.rhel83:6:0: W0611: Unused import osbuild.api (unused-import)
```

Reply:
Similarly, the `osbuild.api` import is now redundant and can be dropped (in both the `rhel82` and `rhel82` runners (which probably is a symlink)).